extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1458,9 +1496,14 @@ static public function onArticleSaveComplete( &$article, &$user, $text, $summary
 		return true;
 	}
 
-	static public function onAllowNotifyOnPageChange( $editor, $title ) {
-		$app = F::app();
-		if ( in_array( MWNamespace::getSubject( $title->getNamespace() ), $app->wg->WallNS ) || $title->getNamespace() == NS_USER_WALL_MESSAGE_GREETING ) {
+	/**
+	 * Don't send email for Wall changes
+	 * @param User $editor
+	 * @param Title $title
+	 * @return bool
+	 */
+	static public function onAllowNotifyOnPageChange( User $editor, Title $title ) {
+		if ( in_array( MWNamespace::getSubject( $title->getNamespace() ), F::app()->wg->WallNS ) || $title->getNamespace() == NS_USER_WALL_MESSAGE_GREETING ) {
```

Comment:
This line is getting a bit long (I think we prefer under 120 chars).  This could probably be made more clear as two conditionals:
```php
if ( in_array( MWNamespace::getSubject( $title->getNamespace() ), F::app()->wg->WallNS ) {
    return false;
}

if ( $title->getNamespace() == NS_USER_WALL_MESSAGE_GREETING ) {
    return false;
}
```